### PR TITLE
New package: Nj0be.HandBrake-SVT-AV1-SPY version 20240914-3e0577889

### PR DESCRIPTION
**Pull request by solomoncyj**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/174321)

### CODE REVIEW

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> 2024-09-19 09:15:59.250 [FAIL] Installer failed security check. Url: https://github.com/Nj0be/HandBrake-SVT-AV1-PSY/releases/download/win/HandBrake-
> 20240914-3e0577889-x86_64-Win_GUI.exe Result: 0x80004005

(Automated response - build 895.)

**Comment by solomoncyj** [comment]
https://www.virustotal.com/gui/file-analysis/ZmVmOGI1Y2MwZmRlYWRiY2YzMTY5YTUxYzFmMmQ3MDU6MTcyNjc4NzcxMQ==

**Comment by stephengillie** [comment]
> Url validation error
> 
> * manifests\n\Nj0be\HandBrake-SVT-AV1-SPY\20240914-3e0577889
>   
>   * https://github.com/Nj0be/HandBrake-SVT-AV1-PSY/releases/download/win/HandBrake-20240914-3e0577889-x86_64-Win_GUI.exe
>     
>     * Http status code: NotFound
>   * https://github.com/Nj0be/HandBrake-SVT-AV1-PSY/releases/download/win/HandBrake-20240914-3e0577889-arm64-Win_GUI.exe
>     
>     * Http status code: NotFound

image

Was the package replaced with a newer version?

**Comment by solomoncyj** [comment]
Yup. Their ci runs quite frequently so every 4 days a version is pushed and the ild is pulled. And i did not want to bother with submitting prs every 4 days

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> 2024-09-24 08:35:43.591 [FAIL] Installer failed security check. Url: https://github.com/Nj0be/HandBrake-SVT-AV1-PSY/releases/download/win/HandBrake-
> 20240922-402dd3fd5-x86_64-Win_GUI.exe Result: 0x80004005

(Automated response - build 896.)

File: manifests/n/Nj0be/HandBrake-SVT-AV1-SPY/20240922-402dd3fd5/Nj0be.HandBrake-SVT-AV1-SPY.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Nj0be.HandBrake-SVT-AV1-SPY
+PackageVersion: 20240922-402dd3fd5
+InstallerType: nullsoft
+UpgradeBehavior: install
+ReleaseDate: 2024-09-16
+Installers:
```
Comment (stephengillie) [inline manifests/n/Nj0be/HandBrake-SVT-AV1-SPY/20240922-402dd3fd5/Nj0be.HandBrake-SVT-AV1-SPY.installer.yaml:9]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Manual Validation ended with:
image
image

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
Verify:
image
image

Hi @solomoncyj,

The generic version number might be problematic. I've approved it, to expedite its availability - but should it have a `DisplayVersion`?

**Pull request merged**